content-repo/extra-docs/articles/Default Pack.md

```diff
@@ -0,0 +1,80 @@
+
+## Introduction
+
+The Default pack provides a solution for new users who are still early in their Cortex XSOAR journey, or as a solution for use-cases that you would like to handle but are yet to create content for.
+
+  
+
+We’ve created a playbook that puts together the core elements of handling any kind of incident. With that playbook there’s also a new layout that gives insight into the data that came with the incident together with any new information collected during the investigation of that incident.
+
+  
+  
+
+## Walkthough
+
+### Playbook:
+
+The playbook makes use of existing out-of-the-box, free content in order to investigate the incident.
+
+It has 3 inputs - whether to extract and enrich indicators from the fetched incident (turn on in case you decided to disable [auto-extract](https://xsoar.pan.dev/docs/incidents/incident-types#incident-type-properties_)), and two tags used for allowing or blocking indicators using an External Dynamic List.
```

Comment:
```suggestion
It has 3 inputs - whether to extract and enrich indicators from the fetched incident (turn on in case you decided to disable [auto-extract](https://xsoar.pan.dev/docs/incidents/incident-types#incident-type-properties)), and two tags used for allowing or blocking indicators using an External Dynamic List.
```
It is not clear to me what the 3 inputs are and what the tags are.

Reply:
I will explain it better